internal/report/json/json.go

```diff
@@ -79,3 +86,52 @@ func (r *Reporter) Reset() {
 func (r *Reporter) ArtifactRequirements() []report.ArtifactType {
 	return nil
 }
+
+func (r *Reporter) buildData() {
+	if len(r.Results) < 1 {
+		return
+	}
+
+	var vdcJobURL string
+	var rdcJobURL string
+	for _, result := range r.Results {
+		if !result.RDC && result.URL != "" {
+			vdcJobURL = result.URL
+			break
+		}
+	}
+	for _, result := range r.Results {
+		if result.RDC && result.URL != "" {
+			rdcJobURL = result.URL
+			break
+		}
+	}
+	vdcBuildURL := r.getBuildURL(vdcJobURL, build.VDC)
+	rdcBuildURL := r.getBuildURL(rdcJobURL, build.RDC)
+	for i, result := range r.Results {
+		if !result.RDC {
+			result.BuildURL = vdcBuildURL
+		} else {
+			result.BuildURL = rdcBuildURL
+		}
+		r.Results[i] = result
+	}
+}
+
+func (r *Reporter) getBuildURL(jobURL string, buildSource build.Source) string {
+	pURL, err := url.Parse(jobURL)
+	if err != nil {
+		log.Debug().Err(err).Msgf("Failed to parse job url (%s)", jobURL)
```

Comment:
`Debug()` is redundant (this goes for the other log statements you have here too).